packages/app/components/Queue/GroupQuestions.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useMemo } from "react";
+import styled from "styled-components";
+import { Card, Row, Col, Checkbox, Collapse, Button } from "antd";
+import { UserAddOutlined } from "@ant-design/icons";
+
+const { Panel } = Collapse;
+
+const QuestionCard = styled(Card)`
+  background: #fafafa;
+  border: 1px solid #d9d9d9;
+  border-radius: 2px;
+  margin-bottom: 8px;
+`;
+
+const QuestionText = styled.div`
+  font-size: 14px;
+  line-height: 22px;
+  color: #595959;
+`;
+
+const AddButton = styled(Button)`
+  margin-top: 8px;
+`;
+
+const CenterRow = styled(Row)`
+  align-items: center;
+  padding-right: 4px;
+`;
+
+const GroupQuestions = () => {
+  return useMemo(() => {
+    return (
+      <Collapse>
```

Comment:
Just curious, would we ever need a max hight to be set on the `Collapse` component?

Reply:
The height depends on how many questions there are in the queue, but it might be cool to make it a scrollable list of questions to save space. I'm not sure which UI is preferable, but I played around a bit with css and it's a little weird trying to hide the scrollbars within ant design. Maybe this could be pushed off for later?

Reply:
Yeah a scrollable list would be cool but it’s definitely not a top priority right now so we can revisit it later